source/main/modules/window-manager/index.ts

```diff
@@ -15,15 +15,17 @@
  * END HEADER
  */
 
-import {
+ import {
```

Comment:
Fails `yarn lint` : `   18:1   error    Expected indentation of 0 spaces but found 1  @typescript-eslint/indent`